tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -824,5 +830,39 @@ int main(int Ac, char **Av) {
                 << ", size in bytes = " << SpecConst.Size
                 << ", type = " << SpecConst.Type << "\n";
   }
+
+  if (SPIRVPrintReport) {
+    std::ifstream IFS(InputFile, std::ios::binary);
+    int ErrCode = 0;
+    std::optional<SPIRV::SPIRVModuleReport> BinReport =
+        SPIRV::getSpirvReport(IFS, ErrCode);
+    if (!BinReport) {
+      std::cout << "Invalid SPIR-V binary, error code is " << ErrCode << "\n";
```

Comment:
Will it be possible to print out a human-readable ErrCode info?

Thanks

Reply:
Very good point indeed, thank you. It would certainly make sense to add human-readable error info. However, the library doesn't support conversion from an error code into a human-readable explanation yet. This could be done if we added one more API call for implementing this conversion. So our choice here basically either to keep this PR as is in the part of reporting error code, or to add the third API call, something like:

`/// \brief Returns the message associated with the error code.
/// \returns empty string if no known error code is found.
std::string getErrorMessage(int ErrCode);`

@asudarsa @MrSidims What would you prefer in this case? To add getErrorMessage() as a part of this PR, or not?

Reply:
ok, thank you

Reply:
Please add a TODO here. Thanks

Reply:
Thank you. I created https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2304 to cover another API call of getErrorMessage(int ErrCode), so that we are able to resolve it quickly after this PR get merged.